x-pack/filebeat/input/s3/config.go

```diff
@@ -18,6 +18,7 @@ type config struct {
 	VisibilityTimeout         time.Duration       `config:"visibility_timeout"`
 	AwsConfig                 awscommon.ConfigAWS `config:",inline"`
 	ExpandEventListFromField  string              `config:"expand_event_list_from_field"`
+	AwsAPITimeout             time.Duration       `config:"aws_api_timeout"`
```

Comment:
Nit. AWS can a bit redundant here :slightly_smiling_face: 
```suggestion
	APITimeout             time.Duration       `config:"api_timeout"`
```